docs/design_guide.rst

```diff
@@ -188,7 +188,7 @@ Design for compatibility with CPython
 
 CircuitPython is aimed to be one's first experience with code.  It will be the
 first step into the world of hardware and software. To ease one's exploration
-out from this first step, make sure that functionality shared with CPython shares
+out from this first step, make sure that the functionality shared with CPython shares
```

Comment:
in this case, we mean "functionality [in general] shared with CPython", and the omission of "the" is fine.
 ```suggestion
out from this first step, make sure that functionality shared with CPython shares
```